python/private/python.bzl

```
@@ -789,6 +796,15 @@ class.
             mandatory = False,
             doc = "The URL template to fetch releases for this Python version. See {attr}`python.single_version_platform_override.urls` for documentation.",
         ),
+        "flag_values": attr.string_dict(
```

This should be the following type.
```suggestion
        "flag_values": attr.label_keyed_string_dict(
```